packages/main/src/ComboBox.js

```diff
@@ -839,9 +839,21 @@ class ComboBox extends UI5Element {
 			this.filterValue = this._selectedItemText;
 		}
 
+		if (isPhone() && event && event.target.getAttribute("icon") === "decline") {
+			this.value = this._lastValue;
+			this.filterValue = this._lastValue;
+		}
+
+		if (isPhone() && this.value !== this._lastValue) {
+			this._fireChangeEvent();
```

Comment:
Please, check the fireChangeEvent method -> it contains the same condition this.value !== this._lastValue, why do we need to check it here again?